RFC-007-SWAP-Basic-Ether.md

```diff
@@ -170,18 +170,18 @@ refund:
 The following is the contract deployment code template encoded as hex which will deploy the above contract when the placeholder values (set to `00`s) are replaced with the parameters.
```

Comment:
:mega: Change this sentence because they are no longer just replaced by `00`s. Say they're replaced by arbitrary values?

Reply:
I adapted the sentence